plugins/notifications/src/hooks/useWebNotifications.ts

```diff
@@ -64,14 +62,14 @@ export function useWebNotifications(enabled: boolean) {
             read: true,
           });
         } else {
-          navigate(notificationsRoute());
+          window.open(notificationsRoute, '_blank');
```

Comment:
I know that this is a controversial topic, but I really dislike `_blank` overall. Besides, this is explicitly going to be in-backstage links right, do we _really_ want those force-opened in a new window? Is this change unrelated to the other fix?

Reply:
I think _blank is not correct here, true.

Reply:
but wasn't also `navigate` the right thing?

Reply:
Yep but having that in the deps also causes hook to trigger :/